api/app/tests/apps/api/v1/test_private_category_endpoint.py

```diff
@@ -58,6 +59,30 @@ def _assert_category_data(self, category, data):
             self.assertEqual(data['sla']['n_days'], slo.n_days)
             self.assertEqual(data['sla']['use_calendar_days'], slo.use_calendar_days)
 
+        self.assertIn('departments', data)
+        if category.departments.count() > 0:
+            self.assertEqual(category.departments.count(), len(data['departments']))
+            category_departments = CategoryDepartment.objects.filter(
+                category_id=category.pk
+            ).order_by(
+                'department__code'
+            )
+            item = 0  # we start with item 0 in the response data
+            for category_department in category_departments:
```

Comment:
Use `enumerate` here + possibly spend a comment on explaining this loop (simultaneously over queryset and API output).

Reply:
Done